docs/Installation.md

```diff
@@ -0,0 +1,101 @@
+2. Expose the ClusterLink deployment to a public IP.
+3. Optionally, create a central gwctl component to control one or more gateways.
+
+Before you begin, please build the project according to the [instructions](../README.md#building-clustelink).
+### 1. ClusterLink deployment and certificates
+In this step, we create the ClusterLink certificates and deployment files.
+
+1) Create a folder (eg. DEPLOY_DIR) for all deployments and yaml files:
+
+        export PROJECT_DIR=`git rev-parse --show-toplevel`
+        export DEPLOY_DIR=~/clusterlink-deployment
+        mkdir -p $DEPLOY_DIR
+        cd $DEPLOY_DIR
+
+2) Create Fabric certificates, including the Root CA certificate for all the clusters (This step needs to be done only once per fabric).:
+
+        $PROJECT_DIR/bin/cl-adm create fabric
+
+    Note: This step needs to be done only once per fabric.
+3) Create certificates and a deployment file for the cluster (e.g., peer1):
+
+        $PROJECT_DIR/bin/cl-adm create peer --name peer1
+
+    Note: By default, the images are pulled from ghcr.io/clusterlink-net. To change the container registry, you can use the ```--container-registry``` flag.  
+    This step needs to be performed for every K8s cluster.
+
+4) Apply ClusterLink deployment:
+   
+        kubectl apply -f $DEPLOY_DIR/peer1/k8s.yaml
+
+5) Verify that all components (cl-controlplane, cl-dataplane, gwctl) are set up and ready.
+   
+        kubectl rollout status deployment cl-controlplane
+        kubectl rollout status deployment cl-dataplane
+        kubectl wait --for=condition=ready pod -l app=gwctl 
+
+### 2. Expose the ClusterLink deployment to a public IP
+Create a public IP for accessing the ClusterLink gateway.
+For a local environment create a K8s nodeport service:
+
+        kubectl apply -f $PROJECT_DIR/demos/utils/manifests/kind/cl-svc.yaml
+        export PEER1_IP=`kubectl get nodes -o "jsonpath={.items[0].status.addresses[0].address}"`
+        export PEER1_PORT=443
+
+For a remote environment:
+
+1. First, create a K8s LoadBalancer service:
+    ```
+    kubectl expose deployment cl-dataplane --name=cl-dataplane-load-balancer --port=443 --target-port=443 --type=LoadBalancer
+    ```
+
+2. Retrieve the LoadBalancer IP when it is allocated:
+    ```
+    export PEER1_IP=$(kubectl get svc -l app=cl-dataplane -o jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+    export PEER1_PORT=443
+    ```
+
+Now, the ClusterLink gateway can be accessed through `$PEER1_IP` at port  `$PEER1_PORT`.
+
+### 3. Create a central gwctl (optional)
+By default for each K8s cluster a gwctl pod is created that use REST APIs to send control messages to the
+ClusterLink gateway, using the command:
+
+    kubectl exec -i <gwctl_pod> -- gwctl <command>
+
+To create a single gwctl that controls one or more ClusterLink gateways, follow these steps:
+
+1. Install the local control (gwctl):
+
+    ```
+    sudo make install
```

Comment:
I think installing into `/usr` is too intrusive and would not recommend it (I plan on removing this make target at a later stage). It can be anywhere in the path (e.g., if using Go then `$GOPATH/bin` is a better location...)

Reply:
Ok but we still need to run this command